gradle-plugin-testing/src/test/java/com/palantir/gradle/plugintesting/PluginTestingJunitPluginTest.java

```diff
@@ -238,6 +238,34 @@ void nested_test() throws Exception {
             }
             """);
 
+        rootProject.testSourceSet().java().writeClass("""
+            package test;
+
+            import com.palantir.gradle.testing.junit.GradlePluginTests;
+            import com.palantir.gradle.testing.junit.DisabledConfigurationCache;
+            import java.nio.file.Files;
+            import org.junit.jupiter.api.Nested;
+            import org.junit.jupiter.api.Test;
+
+            @DisabledConfigurationCache
```

Comment:
do you want to put DisabledConfigurationCache after GradlePluginTests since you had another PR that positions it that way?